2022-02-03 15:22:41,642 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-02-03 15:23:37,178 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55617)
2022-02-03 15:23:37,179 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1643891017.1789103, 'user': {'account_name': 'Guest'}}
2022-02-03 15:23:37,180 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-03 15:23:37,180 DEBUG server.py Соединение с клиентом ('127.0.0.1', 55617) закрывается.
2022-02-03 15:27:57,807 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:27:57,807 ERROR server_log_config.py Ошибка
2022-02-03 15:27:57,807 WARNING server_log_config.py Предупреждение!
2022-02-03 15:27:57,807 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:27:57,807 INFO server_log_config.py Информационное сообщение
2022-02-03 15:29:21,961 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:29:21,961 ERROR server_log_config.py Ошибка
2022-02-03 15:29:21,961 WARNING server_log_config.py Предупреждение!
2022-02-03 15:29:21,961 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:29:21,961 INFO server_log_config.py Информационное сообщение
2022-02-03 15:29:57,499 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:29:57,499 ERROR server_log_config.py Ошибка
2022-02-03 15:29:57,499 WARNING server_log_config.py Предупреждение!
2022-02-03 15:29:57,499 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:29:57,499 INFO server_log_config.py Информационное сообщение
2022-02-03 15:31:45,313 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:31:45,313 ERROR server_log_config.py Ошибка
2022-02-03 15:31:45,313 WARNING server_log_config.py Предупреждение!
2022-02-03 15:31:45,313 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:31:45,313 INFO server_log_config.py Информационное сообщение
2022-02-03 15:32:28,672 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:32:28,672 ERROR server_log_config.py Ошибка
2022-02-03 15:32:28,672 WARNING server_log_config.py Предупреждение!
2022-02-03 15:32:28,672 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:32:28,672 INFO server_log_config.py Информационное сообщение
2022-02-03 15:33:28,417 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-02-03 15:33:30,718 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56051)
2022-02-03 15:33:30,719 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1643891610.7183607, 'user': {'account_name': 'Guest'}}
2022-02-03 15:33:30,720 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-03 15:33:30,720 DEBUG server.py Соединение с клиентом ('127.0.0.1', 56051) закрывается.
2022-02-03 15:38:12,434 INFO server.py Установлено соедение с ПК ('127.0.0.1', 63308)
2022-02-03 15:38:12,435 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1643891892.4330306, 'user': {'account_name': 'Guest'}}
2022-02-03 15:38:12,438 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-03 15:38:12,438 DEBUG server.py Соединение с клиентом ('127.0.0.1', 63308) закрывается.
2022-02-03 15:38:46,030 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:38:46,030 ERROR server_log_config.py Ошибка
2022-02-03 15:38:46,030 WARNING server_log_config.py Предупреждение!
2022-02-03 15:38:46,030 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:38:46,030 INFO server_log_config.py Информационное сообщение
2022-02-03 15:38:56,293 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-02-03 15:39:01,493 INFO server.py Установлено соедение с ПК ('127.0.0.1', 63359)
2022-02-03 15:39:01,494 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1643891941.4933116, 'user': {'account_name': 'Guest'}}
2022-02-03 15:39:01,495 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-03 15:39:01,495 DEBUG server.py Соединение с клиентом ('127.0.0.1', 63359) закрывается.
2022-02-03 15:39:05,539 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:39:05,539 ERROR server_log_config.py Ошибка
2022-02-03 15:39:05,539 WARNING server_log_config.py Предупреждение!
2022-02-03 15:39:05,539 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:39:05,539 INFO server_log_config.py Информационное сообщение
2022-02-03 15:44:00,673 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:44:00,677 ERROR server_log_config.py Ошибка
2022-02-03 15:44:00,677 WARNING server_log_config.py Предупреждение!
2022-02-03 15:44:00,677 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:44:00,677 INFO server_log_config.py Информационное сообщение
2022-02-03 15:44:09,919 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-02-03 15:44:16,067 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56685)
2022-02-03 15:44:16,068 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1643892256.0673692, 'user': {'account_name': 'Guest'}}
2022-02-03 15:44:16,070 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-03 15:44:16,070 DEBUG server.py Соединение с клиентом ('127.0.0.1', 56685) закрывается.
2022-02-03 15:44:20,027 CRITICAL server_log_config.py Критическая ошибка
2022-02-03 15:44:20,028 ERROR server_log_config.py Ошибка
2022-02-03 15:44:20,028 WARNING server_log_config.py Предупреждение!
2022-02-03 15:44:20,028 DEBUG server_log_config.py Отладочная информация
2022-02-03 15:44:20,028 INFO server_log_config.py Информационное сообщение
